src/resources/js/tickets-block.js

```diff
@@ -694,6 +694,12 @@ tribe.tickets.block = {
 
 		const s = ( prec ? toFixedFix( n, prec ) : Math.round( n ) ).toString().split( dec );
```

Comment:
```suggestion
		let s = ( prec ? toFixedFix( n, prec ) : Math.round( n ) ).toString().split( dec );
```

Reply:
good catch for some reason I glossed over the const part.....